com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Camera/HDAdditionalCameraEditor.cs

```diff
@@ -9,5 +9,11 @@ class HDAdditionalCameraDataEditor : Editor
         public override void OnInspectorGUI()
         {
         }
+
+        [MenuItem("CONTEXT/HDAdditionalCameraData/Remove Component")]
+        static void RemoveComponent(MenuCommand command)
```

Comment:
Can we unify names? On light it is `RemoveAdditionalComponent` and on Camera it is `RemoveComponent`